docs/content/3.middleware/2.guides/10.working-with-logger.md

```diff
@@ -0,0 +1,311 @@
+# Working with logger
+
+The middleware application provides a logger instance that automatically attaches metadata related to the scope of each call. This scope includes crucial information such as the name of the integration, the extension (if applicable), the function name (if applicable), the hook name (if applicable), and the type of the action being logged (e.g., endpoint, bootstrapHook, requestHook). By adding this contextual data, the logger makes it significantly easier to trace the origin of logs or errors, simplifying the debugging and monitoring process across the application.
```

Comment:
I wonder if we have be that specific here? Specific information are usually outdated fast and must be adjusted. Maybe, instead of listing what we add, we just describe in general what we add? (no details)